### PR TITLE
Added footer translator comment

### PR DESCRIPTION
**Pull request by khleomix**
Amendment for task #171 (see PR #188)

@kjellr @grappler please check if this works. Thanks

### CODE REVIEW

**Comment by khleomix** [comment]
@grappler got it, was wondering about that as well. Will have to get back to this tomorrow though. Will update. Thanks!

**Comment by khleomix** [comment]
@grappler please check if I've covered everything. Thanks

File: footer.php [hunk 1]
```diff
@@ -18,6 +18,7 @@
 		<div class="site-info">
 			<a class="site-name" href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></a>,
 			<a href="<?php echo esc_url( __( 'https://wordpress.org/', 'twentynineteen' ) ); ?>" class="imprint">
+				<?php /* translators: %s: CMS name, i.e. WordPress. */ ?>
 				<?php printf( __( 'Proudly powered by %s', 'twentynineteen' ), 'WordPress' ); ?>.
```
Comment (grappler) [inline footer.php]: Now that I am looking at the code again I am wondering why the string can't be just "Proudly powered by WordPress". It is not like this theme can be installed in another CMS.

File: inc/template-tags.php [hunk 2]
```diff
@@ -11,8 +11,10 @@
 	 * Prints HTML with meta information for the current post-date/time.
 	 */
 	function twentynineteen_posted_on() {
+		/* translators: 1: post date in Full Date/Time. 2: post date in F j, Y. */
 		$time_string = '<time class="entry-date published updated" datetime="%1$s">%2$s</time>';
```
Comment (grappler) [inline inc/template-tags.php]: There is no internationalization function used here the translator comment is not needed.

File: inc/template-tags.php [hunk 3]
```diff
@@ -11,8 +11,10 @@
...
 		if ( get_the_time( 'U' ) !== get_the_modified_time( 'U' ) ) {
+			/* translators: 1: post date in Full Date/Time. 2: post date in F j, Y. 3: updated post date in Full Date/Time. 2: updated post date in F j Y.  */
 			$time_string = '<time class="entry-date published" datetime="%1$s">%2$s</time><time class="updated" datetime="%3$s">%4$s</time>';
```
Comment (grappler) [inline inc/template-tags.php]: Same here.

File: inc/template-tags.php [hunk 4]
```diff
@@ -25,6 +27,7 @@ function twentynineteen_posted_on() {
 		);
 
 		printf(
+			/* translators: 1: SVG icon, 2: link to post */
 			'<span class="posted-on">%1$s<a href="%2$s" rel="bookmark">' . $time_string . '</a></span>',
```
Comment (grappler) [inline inc/template-tags.php]: Same.

File: inc/template-tags.php [hunk 5]
```diff
@@ -39,8 +42,8 @@ function twentynineteen_posted_on() {
 	 */
 	function twentynineteen_posted_by() {
 		printf(
-			'<span class="byline">%1$s<span class="screen-reader-text">%2$s</span><span class="author vcard"><a class="url fn n" href="%3$s">%4$s</a></span></span>',
 			/* translators: 1: SVG icon. 2: post author, only visible to screen readers. 3: author link. */
+			'<span class="byline">%1$s<span class="screen-reader-text">%2$s</span><span class="author vcard"><a class="url fn n" href="%3$s">%4$s</a></span></span>',
```
Comment (grappler) [inline inc/template-tags.php]: Same.

File: template-parts/post/author-info.php [hunk 6]
```diff
@@ -9,6 +9,7 @@
 if ( (bool) get_the_author_meta( 'description' ) ) : ?>
 <div class="author-description">
 	<h2 class="author-title">
+		<?php /* translators: %s: post author */ ?>
 		<span class="author-heading"><?php echo esc_html( sprintf( __( 'Published by %s', 'twentynineteen' ), get_the_author() ) ); ?></span>
```
Comment (grappler) [inline template-parts/post/author-info.php]: Looks good. Though I would have written the code as

```php
printf(
	/* translators: %s: post author */
	__( 'Published by %s', 'twentynineteen' ),
	esc_html( get_the_author() )
);
```

File: footer.php (hunk 1 above)
Comment (khleomix) [reply footer.php]: @grappler You're right, I copied that over from the Gutenberg Starter Theme (https://github.com/WordPress/gutenberg-starter-theme/blob/master/footer.php) will update and fix as per your comments.

**Comment by khleomix** [comment]
@grappler please recheck and let me know if there are more changes.

File: footer.php (hunk 1 above)
Comment (grappler) [reply footer.php]: Sorry, I meant the code should be
```php
_e( 'Proudly powered by WordPress', 'twentynineteen' )
```

Comment (khleomix) [reply footer.php]: no worries :) added.

File: template-parts/post/discussion-meta.php [hunk 7]
```diff
@@ -17,12 +17,15 @@
 	/* translators: %1(X responses)$s from %2(X others)$s */
 	$meta_label = sprintf(
 		'%1$s from %2$s.',
+		/* translators: %d(X responses)$s */
 		sprintf( _n( '%d response', '%d responses', $discussion->responses, 'twentynineteen' ), $discussion->responses ),
+		/* translators: %d(X others)$s */
 		sprintf( _n( '%d other', '%d others', $discussion->commenters, 'twentynineteen' ), $discussion->commenters )
```
Comment (dimadin) [inline template-parts/post/discussion-meta.php]: See #218.

**Pull request merged**